### PR TITLE
Fixed : Spacing for Simple APIs

### PR DESCRIPTION
**Pull request by akar016012**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Bug fix, feature, docs update, ...

Bug Fix

## What is the current behavior?

image

## What is the new behavior?

image

## Additional context

added mt-16 to add a top margin so that the content is not cutoff

### CODE REVIEW

**Review by fsansalvadore — CHANGES_REQUESTED** [review]
Thanks @akar016012, I left a few suggestions.

File: apps/www/pages/database/Database.tsx [hunk 1]
```diff
@@ -234,7 +234,7 @@ function Database() {
           </div>
         </Swiper>
 
-        <SectionContainer className="-mb-48">
+        <SectionContainer className="mb-48">
```
[the remark this answers is not in the document]
Comment (akar016012) [reply apps/www/pages/database/Database.tsx:237]: no its not!
image

**Comment by akar016012** [comment]
> Thanks @akar016012, I left a few suggestions.

Thank you, very helpful! 💯

File: apps/www/pages/database/Database.tsx [hunk 2]
```diff
@@ -234,7 +234,7 @@ function Database() {
           </div>
         </Swiper>
 
-        <SectionContainer className="-mb-48">
+        <SectionContainer>
```
[the remark this answers is not in the document]
Comment (fsansalvadore) [reply apps/www/pages/database/Database.tsx:237]: This was fine actually, lets stick for the Auth page for now 👍

**Review by fsansalvadore — APPROVED** [review]

**Comment by fsansalvadore** [comment]
Merged, thanks @akar016012 🎉

**Comment by fsansalvadore** [comment]
#26392

**Pull request merged**